errbot/plugin_wizard.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+import configparser
+import os
+import re
+import requests
+import sys
+
+from errbot import PY2
+
+if PY2:
+    input = raw_input
+
+
+def new_plugin_wizard(directory=None):
+    """
+    Start the wizard to create a new plugin in the current working directory.
+    """
+    if directory is None:
+        print("This wizard will create a new plugin for you in the current directory.")
+        directory = os.getcwd()
+    else:
+        print("This wizard will create a new plugin for you in '%s'." % directory)
+
+    if not os.path.isdir(directory):
+        print("Error: The path '%s' does not exist or is not a directory." % directory)
+        sys.exit(1)
+
+    try:
+        plugin_code = download_plugin_skeleton()
+    except requests.HTTPError as e:
+        print(
+            "Error: I couldn't download the plugin skeleton from GitHub "
+            "which is needed to write out a new plugin file for you."
+        )
+        print("Details: {!s}".format(e))
+        sys.exit(1)
+
+    name = ask(
+        "What should the name of your new plugin be?",
+        validation_regex=r'^[a-zA-Z][a-zA-Z0-9 _-]*$'
+    ).strip()
+    module_name = name.lower().replace(' ', '_')
+    class_name = "".join([s.capitalize() for s in name.lower().split(' ')])
+
+    description = ask(
+        "What may I use as a short (one-line) description of your plugin?"
+    )
+    python_version = ask(
+        "Which python version will your plugin work with? 2, 2+ or 3?",
+        valid_responses=['2', '2+', '3']
```

Comment:
I considered this, but I'd prefer to let the user really think about which Python versions they want to support, instead of automatically defaulting to what they're currently using to develop on.